src/explore-education-statistics-admin/src/pages/release/utils/releaseStatusDetail.ts

```diff
@@ -0,0 +1,25 @@
+import { StatusDetail } from '@admin/pages/release/hooks/useReleasePublishingStatus';
+
+export default function getStatusDetail(status: string): StatusDetail {
```

Comment:
When we only have a single default export for a file, it makes sense to name the file accordingly to i.e. `getStatusDetail.tsx`